apps/mobile/src/components/Comments/Comments.js

```diff
@@ -82,66 +62,64 @@ export const Comments = React.forwardRef(({
     if (!comment) return null
+    // Inverted flat list so comments will render in reverse order, but child comments
+    // here will not, and the query is setup to query in desc order. Could be changed to "asc"
+    // but going to keep it has been for now.
+    const reversedChildComments = [...comment.childComments.items].reverse()
     return (
       <>
-        <ShowMore postOrComment={comment} style={styles.childCommentsShowMore} />
         <Comment
           clearHighlighted={() => setHighlightedComment(null)}
           comment={comment}
           groupId={groupId}
-          highlighted={comment?.id === highlightedComment?.id}
+          highlighted={comment.id === highlightedComment?.id}
           onReply={selectComment}
           postTitle={post?.title}
           scrollTo={viewPosition => scrollToComment(comment, viewPosition)}
           setHighlighted={() => setHighlightedComment(comment)}
           showMember={showMember}
           key={comment.id}
         />
+        <ShowMore postOrComment={comment} style={styles.childCommentsShowMore} />
+        {reversedChildComments.map(childComment => (
+          <Comment
+            clearHighlighted={() => setHighlightedComment(null)}
+            comment={childComment}
+            groupId={groupId}
+            highlighted={childComment.id === highlightedComment?.id}
+            onReply={selectComment}
+            postTitle={post?.title}
+            scrollTo={viewPosition => scrollToComment(childComment, viewPosition)}
+            setHighlighted={() => setHighlightedComment(childComment)}
+            showMember={showMember}
+            style={styles.childComment}
+            key={childComment.id}
+          />
+        ))}
       </>
     )
   }
 
-  // comment.childComments rendering
-  const Item = ({ item: comment }) => {
-    if (!comment) return null
-    return (
-      <Comment
-        clearHighlighted={() => setHighlightedComment(null)}
-        comment={comment}
-        groupId={groupId}
-        highlighted={comment.id === highlightedComment?.id}
-        onReply={selectComment}
-        postTitle={post?.title}
-        scrollTo={viewPosition => scrollToComment(comment, viewPosition)}
-        setHighlighted={() => setHighlightedComment(comment)}
-        showMember={showMember}
-        style={styles.childComment}
-        key={comment.id}
-      />
-    )
-  }
-
   if (fetching) {
     return <Loading />
   }
 
   return (
-    <SectionList
+    <FlashList
       style={style}
       contentContainerStyle={styles.contentContainerStyle}
       ref={commentsListRef}
       // Footer is Header, etc.
       inverted
       ListFooterComponent={Header}
-      renderSectionFooter={SectionFooter}
-      renderItem={Item}
-      sections={sections}
+      renderItem={renderItem}
+      data={comments}
+      // This means that FlashList will re-render anytime this/these values change
+      extraData={highlightedComment}
```

Comment:
FYI -- if we find any quirks in critical places, likely it will be around other extraData things needing to be added. For now I don't see others. extraData is basically equal to the dependency array in a hook but for items in the list.